python/src/scipp/_variable.py

```diff
@@ -194,8 +194,7 @@ def array(*,
                     dtype=dtype)
 
 
-def linspace(*,
-             dim: str,
+def linspace(dim: str,
              start: Union[int, float],
              stop: Union[int, float],
              num: int,
```

Comment:
Now this seems to support non-keyword args for all args? I am not sure that is a good approach?

Reply:
ah sorry, should I add the `*` before the scipp arguments?

Reply:
Not sure, see discussion I tried to start with https://github.com/scipp/scipp/pull/1871#issuecomment-827488778.

Reply:
I think it's good like that, i think definitely people who use numpy are almost never using kwargs in these functions